lib/web_console/request.rb

```diff
@@ -32,6 +32,49 @@ def acceptable_content_type?
       content_type.blank? || content_type.in?(acceptable_content_types)
     end
 
+    # Returns whether the request includes "application/vnd.web-console"
+    def has_media_type?
+      accept.split(/\s*;\s*/, 2)[0].split(/\s*,\s*/).any? do |t|
+        /^application\/vnd.web-console$/ === t
+      end
+    end
+
+    # Returns whether the request requires versioning in accept header
+    def has_version_requirement?
+      ! required_version.nil?
+    end
+
+    # Returns whether the request is acceptable
+    #
+    # It returns true if required version and current version have
+    # the same major and minor version.
+    # It also returns true if a request includes no vendor media type.
+    def acceptable_required_version?
```

Comment:
@gsamokovarov Thanks for the reviews. Yeah, I totally agree with your points.

> If we utilize the Mime module, we won't need to do all this parsing by ourselves. See the implementation of ActionDispatch::Request#accepts, if we register our own MIME Type trough it, we get parsing for free.

That seems great. I will rewrite code with the module later.